docs_src/microservices/device/supported/device-onvif-camera/Walkthrough/deployment.md

```diff
@@ -10,27 +10,37 @@ Follow this guide to deploy and run the service.
          ```bash
          cd edgex-compose/compose-builder/
          ```
+      2. Run Edgex with the ONVIF microservice in secure or non-secure mode.
 
-      2. Run EdgeX with the microservice in non-secure mode:
+        ##### Non-secure mode
 
-         ```bash
-         make run no-secty ds-onvif-camera
-         ```
-      
-      3. Run EdgeX with the microservice in secure mode:
+        ```bash
+        make run no-secty ds-onvif-camera
+        ```
+    
+        ##### Secure mode 
+
+        !!! note
+            Recommended for secure and production level deployments. Make a note of the Consul ACL token and the JWT token generated 
+            which are needed to map credentials and execute apis.
 
          ```bash
          make run ds-onvif-camera
+         make get-consul-acl-token
```

Comment:
these commands are not ready to run right away. may want to instruct the user to wait a bit before running these.